data/campaigns/Heir_To_The_Throne/achievements.cfg

```diff
@@ -1 +1,552 @@
 #textdomain wesnoth-httt
+
+[achievement_group]
+    display_name=_"Heir to the Throne"
+    content_for=Heir_To_The_Throne_achievements
+
+    [achievement]
+        id="HttT_S1_Defend_Aethenwood"
+        name=_"Defend Aethenwood from the Orcs"
+        name_completed= _ "Defended Aethenwood from the Orcs"
+        description=_"Defeat any two of the three enemy orcish commanders."
+        description_completed=_"Slain two of the three enemy orcish commanders."
+        icon="units/elves-wood/captain-defend.png~RC(magenta>red)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+        icon_completed="units/elves-wood/captain-defend.png~RC(magenta>red)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+    [/achievement]
+
+    # Note: you can get this but you lose the scenario
+    [achievement]
+        id="HttT_S1_Broken_Offensive"
+        name=_"Broken Offensive"
+        name_completed= _ "Broken Offensive"
+        description=_"Defeat all three enemy orcish commanders."
+        description_completed=_"Slain all three enemy orcish commanders."
+        icon="units/elves-wood/captain-defend.png~RC(magenta>red)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+        icon_completed="units/elves-wood/captain-defend.png~RC(magenta>red)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+    [/achievement]
+
+    [achievement]
+        id="HttT_S2_Guardian_of_the_Blackwater"
+        name=_"Guardian of the Blackwater"
+        name_completed=_"Guardian of the Blackwater"
+        description=_"Defeat the enemy leader in <i>Blackwater Port</i>"
```

Comment:
So, one general consistency question: should achievement descriptions end in a period or not? I'm seeing it both ways throughout the PR...